File: fix_app.js
const fs = require('fs');
const path = 'd:\\Aplikasi\\APP\\DAKU\\App.tsx';

try {
    const content = fs.readFileSync(path, 'utf8');
    // Handle potentially different line endings by splitting on \n and cleaning \r later if needed, 
    // but usually split(/\r?\n/) is safest.
    const lines = content.split(/\r?\n/);

    // Target: Remove lines 564 to 606 (1-based indices)
    // Array indices: 563 to 605
    // Count: 606 - 564 + 1 = 43 lines.

    // Verification
    console.log(`Deleting block start (Line 564): ${lines[563]}`);
    console.log(`Deleting block end (Line 606): ${lines[605]}`);

    // splice(start, deleteCount)
    lines.splice(563, 43);

    // Join back with standard newline
    const newContent = lines.join('\n');
    fs.writeFileSync(path, newContent, 'utf8');
    console.log('App.tsx cleaned successfully.');
} catch (e) {
    console.error('Error:', e);
    process.exit(1);
}

File: fix_app.py
import os

file_path = r'd:\Aplikasi\APP\DAKU\App.tsx'

with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Removing lines 564 to 606 (1-based)
# Indices: 563 to 605
# Python slice to remove: [563:606]
# Verification:
# lines[563] is the line that starts with /*
# lines[605] is the line that ends with ); */
# lines[606] is the line }

print(f"Deleting lines: {lines[563].strip()} ... {lines[605].strip()}")

del lines[563:606]

with open(file_path, 'w', encoding='utf-8') as f:
    f.writelines(lines)

print("Done.")
